airbyte-cdk/python/setup.py

```diff
@@ -70,7 +70,4 @@
             "sphinx-rtd-theme~=1.0",
         ],
     },
-    entry_points={
```

Comment:
this is technically backwards breaking but i'm 99.999% sure only davin and I used this when we first wrote the CDK